package.json

```diff
@@ -55,13 +55,15 @@
     "express-fileupload": "^1.2.1",
     "express-ipfilter": "^1.2.0",
     "fast-sort": "^3.1.1",
+    "forever": "^4.0.1",
     "fs-extra": "^10.0.0",
     "graphdb": "^2.0.0",
     "it-concat": "^2.0.0",
     "it-length-prefixed": "^5.0.3",
     "it-pipe": "^1.1.0",
     "js-sha3": "^0.8.0",
     "json-2-csv": "^3.14.4",
+    "json-stringify-safe": "^5.0.1",
```

Comment:
Is this necessary?

Reply:
If it's not needed then I can leave it out

Reply:
removed